src/sentry/issues/json_schemas.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any, Mapping
+
+EVENT_PAYLOAD_SCHEMA: Mapping[str, Any] = {
+    "type": "object",
+    "properties": {
+        # required properties
+        "event_id": {"type": "string", "minLength": 1},
+        "platform": {"type": "string", "minLength": 1},
+        "project_id": {"type": "integer"},
+        "tags": {"type": "object"},
+        "timestamp": {"type": "string", "format": "date-time"},
+        "message_timestamp": {"type": "string", "format": "date-time"},
+        # "title": {"type": "string", "minLength": 1}, leaving this out, for now
+        # non-required properties
+        "breadcrumbs": {"type": "object"},
```

Comment:
Does object in js mean specifically a dict? We might want to be a bit more explicit here and defined the format of the breadcrumbs, since it will be an array of breadcrumb dicts: https://develop.sentry.dev/sdk/event-payloads/types/#breadcrumb

Reply:
ah yes. will change to array etc.